native/database/db_test.go

```diff
@@ -0,0 +1,84 @@
+package database
+
+import (
+	"testing"
+)
+
+func TestInsertTransaction(t *testing.T) {
+	err := InitDB(":memory:") // in-memory database
```

Comment:
create a helper method to initialize the database for tests:
```
func initDB(t *testing.T) DB { ... }
```

Reply:
Done